### PR TITLE
Fix activity text not displaying reward total

### PR DESCRIPTION
**Pull request by Duddino**
## Abstract

Fix activity text not displaying reward total
---

## Testing
To test this PR, it's suggested to attempt these user flows, or variations of these:
- Access a wallet
- Check that the rewards tab displays the correct reward total in PIVs

### CODE REVIEW

File: scripts/dashboard/Activity.vue [hunk 1]
```diff
@@ -228,6 +233,13 @@ async function parseTXs(arrTXs) {
 
     txs.value = newTxs;
 }
+if (props.rewards) {
```
Comment (panleone) [inline scripts/dashboard/Activity.vue:236]: maybe move this to the function `update` instead of `parseTXs`?

File: scripts/dashboard/Activity.vue [hunk 2]
```diff
@@ -96,6 +99,7 @@ async function update(txToAdd = 0) {
     await parseTXs(arrTXs);
     txCount = found;
     updating.value = false;
+    rewardsText.value = (mempool.coldBalance / COIN).toFixed(2);
```
Comment (panleone) [inline scripts/dashboard/Activity.vue:102]: remove this line? I don't understand

**Review by panleone — COMMENTED** [review]
working as expected, just 2 comments

File: scripts/dashboard/Activity.vue (hunk 2 above)
Comment (Duddino) [reply scripts/dashboard/Activity.vue]: Oops leftover from testing

File: scripts/dashboard/Activity.vue (hunk 1 above)
Comment (Duddino) [reply scripts/dashboard/Activity.vue:235]: It's not in `parseTXs`, it's at the top level

Comment (panleone) [reply scripts/dashboard/Activity.vue:235]: oooh ok missed that

**Review by panleone — APPROVED** [review]
tACK 6e2615e6cf06ee42d44f6a80b55c1b10e1736e19

**Review by JSKitty — APPROVED** [review]
**tACK** 6e2615e6cf06ee42d44f6a80b55c1b10e1736e19

**Out of scope idea**: I think now that we have the total history; it might be a nice idea for users to see the true **Total** rewards of all history (only the total reward number, not the full TX Activity list), it shouldn't be too computationally expensive, and it's easy to cache. LGTM.

**Comment by Duddino** [comment]
> **Out of scope idea**: I think now that we have the total history; it might be a nice idea for users to see the true **Total** rewards of all history (only the total reward number, not the full TX Activity list), it shouldn't be too computationally expensive, and it's easy to cache. LGTM.

I thought about that too, just wanna keep these bug fixes simple because they're gonna get included to 1.4.1

**Pull request merged**